src/components/staking/components/ValidatorBox.js

```diff
@@ -129,7 +129,7 @@ export default function ValidatorBox({
     const dispatch = useDispatch()
     const { accountId: validatorId, active } = validator
 
-    const fee = validator.fee && validator.fee.percentage
+    const fee = Math.round(validator.fee && validator.fee.percentage * 10) / 10
```

Comment:
best to use https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toFixed to make intent clear